src/smexperiments/experiment.py

```diff
@@ -101,6 +103,7 @@ def create(cls, experiment_name=None, description=None, sagemaker_boto_client=No
             experiment_description: (str, optional): Description of the experiment
             sagemaker_boto_client (SageMaker.Client, optional): Boto3 client for SageMaker. If not
                 supplied, a default boto3 client will be created and used.
+            tags (List[dict[str, str]]): A list of tags to associate with the experiment.
```

Comment:
should just be ``` tags (dict): A dictionary of ....``` List of dict implies there are multiple dictionaries.

Reply:
Based on the tag under create_trial_component https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html?highlight=experiment#SageMaker.Client.create_trial_component, should it be a list of dict?

Reply:
looks like a list of dict is the normal way to represent tags https://github.com/aws/sagemaker-python-sdk/search?p=3&q=tags&unscoped_q=tags so go with it